src/routes/safe/store/actions/createTransaction.ts

```diff
@@ -241,8 +262,7 @@ const createTransaction = ({
           dispatch,
           state,
         )
-
-        dispatch(fetchTransactions(safeAddress))
+        ;(await fetchTransactions(safeAddress))(dispatch)
```

Comment:
what was wrong with the previous syntax?
```dispatch(fetchTransactions(safeAddress))```

Reply:
couldn't use `await` without TS complaining, thus this was my workaround.

Reply:
why did you need to use await?

Reply:
<img width="872" alt="Screenshot 2020-07-15 at 16 39 08" src="https://user-images.githubusercontent.com/16622558/87545883-bec43300-c6b9-11ea-81a7-452b2a42f6d4.png">

doesn't seem like await affects anything